GVFS/GVFS.Common/Git/GitProcess.cs

```diff
@@ -178,13 +178,27 @@ public virtual void RejectCredentials(string repoUrl, string username = null, st
 
             string stdinConfig = sb.ToString();
 
-            this.InvokeGitOutsideEnlistment(
+            Result result = this.InvokeGitOutsideEnlistment(
                 "credential reject",
                 stdin => stdin.Write(stdinConfig),
                 null);
+
+            if (result.ExitCodeIsFailure)
+            {
+                EventMetadata errorData = new EventMetadata();

```

Comment:
Should we be adding anything to this `EventMetadata`?  If not, you could remove it and call the version of `RelatedWarning` that takes a `string` as its only parameter.